src/sweeper/sweepers/duplicates.py

```diff
@@ -7,52 +7,54 @@
 
 import arcpy
 import re
+import os
 from xxhash import xxh64
 
-
 class DuplicateTest(object):
-    def __init__(self):
+    def __init__(self, workspace, table_name):
+        self.report = {}
+        self.workspace = workspace
+        self.table_name = table_name
 
 
-    def sweep(self, fc):
+    def sweep(self):
 
         digestDict = {}
-        duplicateDict = {}
-        dupePrintLst = []
+
         digTrim = re.compile(r'(\d+\.\d{2})(\d+)')
 
-        shps = ['Shape', 'SHAPE']
+        arcpy.env.workspace = self.workspace
```

Comment:
it's a good practice to [reset](https://pro.arcgis.com/en/pro-app/arcpy/functions/clearenvironment.htm) the workspace after setting it so no other code is fouled up later in the process.

Reply:
The environment manager that Steve mentions below is probably a good idea to use here and below